compiler/scenario-service/protos/scenario_service.proto

```diff
@@ -428,6 +429,7 @@ message ScenarioResult {
   sfixed64 final_time = 4;
   repeated TraceMessage trace_log = 5;
   repeated WarningMessage warnings = 6;
+  repeated string active_contracts = 7;
```

Comment:
from a quick look, i think the issue is not on the Haskell side. We’re just not setting that field anywhere in Scala. We need to set this in `Conversions.scala`.